_posts/2022-11-05-rspamd-3.4.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Rspamd 3.4 has been released"
+categories: announce
+---
+
+We have released **Rspamd 3.4** today. This is a bugfix release with no incompatible changes. Several new features have also been added.
+Here are the most important changes in this version explained.
+
+## Main changes
+
+### Sharing hyperscan database among Rspamd processes
+
+Now hyperscan databases are shared between all Rspamd processes reducing memory footprint, especially in the case where many processes are running.
+
+### Critical fix in the compatibility with the integrations and headers alterations
+
+Fixed integration with OpenSMTPD.
+
+### Fix additional fields in the Redis schema
+
+After version 3.2, some fields were no longer accepted in Redis settings. Now it works correctly.
```

Comment:
In fact, that problem was persistent even pre 3.2 (I'm not sure when the redis schema has been introduced, but it was even before 3.0 to my best knowledge). A link to the relevant issue is also appreciated.